README.md

```diff
@@ -1,3 +1,5 @@
+[![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2Fscore-spec%2Fspec.svg?type=shield)](https://app.fossa.com/projects/git%2Bgithub.com%2Fscore-spec%2Fspec?ref=badge_shield) [![OpenSSF Best Practices](https://www.bestpractices.dev/projects/9204/badge)](https://www.bestpractices.dev/projects/9204)
```

Comment:
Might look better if we put them underneath the score banner image?

Reply:
That's the usual convention, for example in Helm https://github.com/helm/helm, python https://github.com/python/cpython, etc.